fr/en-cours/originales/transcription-automatisee-graphies-non-latines.md

```diff
@@ -758,24 +760,24 @@ Tableau 6: Évolution du CER en fonction du nombre d'images transcrites
 
-{% include figure.html filename="figure21_time_saved_transcription.png" caption="Figure 21 : RASAM Dataset, Springer 2021 - Evolution du CER et du temps de relecture" %}
-<!-- ![Figure 21 : RASAM Dataset, Springer 2021 - Evolution du CER et du temps de relecture](figure21_time_saved_transcription.png) -->
+{% include figure.html filename="figure21_time_saved_transcription.png" caption="Figure&nbsp;21 : RASAM Dataset, Springer 2021 - Evolution du CER et du temps de relecture" %}
+<!-- ![Figure&nbsp;21 : RASAM Dataset, Springer 2021 - Evolution du CER et du temps de relecture](figure21_time_saved_transcription.png) -->
 
 Dans ce tutoriel, nous avons décrit les bonnes pratiques pour la transcription rapide de documents en systèmes graphiques ou en langues peu dotés via la plateforme Calfa Vision. La qualification de "peu dotée" peut concerner un grand nombre et une grande variété de documents, y compris, comme ce fut le cas ici, dans des langues pour lesquelles il existe pourtant déjà des données. La qualité des données ne doit pas être négligée par rapport à la quantité, et l'utilisateur pourra dès lors envisager une transcription, y compris pour des documents inédits.
 
 
 <div class="alert alert-warning">
-La stratégie de <i>fine-tuning</i> s'avère très pertinente dans les situations où il n'est pas possible de constituer un jeu de données suffisant, quelque soit le document ou la langue. Néanmoins, il faut prendre garde au fait que les modèles créés via cette stratégie sont dès lors sur-spécialisés sur la problématique cible (en raison de tous les choix éditoriaux présentés). Cette stratégie n'est par ailleurs pas unique : il existe par exemple en apprentissage machine des stratégies reposant sur l'augmentation des données qui pourraient être implémentées, mais qui ne sont pas proposées par les plateformes grand public.
+La stratégie de <i>fine-tuning</i> s'avère très pertinente dans les situations où il n'est pas possible de constituer un jeu de données suffisant, quelque soit le document ou la langue. Néanmoins, il faut prendre garde au fait que les modèles créés via cette stratégie sont dès lors sur-spécialisés sur la problématique cible (en raison de tous les choix éditoriaux présentés). Cette stratégie n'est par ailleurs pas unique : il existe par exemple en apprentissage machine des stratégies reposant sur l'<a href="https://en.wikipedia.org/wiki/Data_augmentation">augmentation des données</a>.
```

Comment:
@CVidalG do you agree with this modification ?

Reply:
But, yes, we do need to use html for formatting and links within <div> info boxes 🙂